apps/storybook/src/Header.stories.tsx

```diff
@@ -165,7 +166,9 @@ export const Full: Story<HeaderProps> = (args) => {
           onClick={() => action('Clicked on the notification bell')()}
           styleType='borderless'
         >
-          <SvgNotification />
+          <NotificationMarker status='positive' pulsing={true}>
+            <SvgNotification />
+          </NotificationMarker>
```

Comment:
The test image that goes along with this story should be updated.

Reply:
**[DONE]**